xml/System.Security.Cryptography/DESCryptoServiceProvider.xml

```diff
@@ -133,9 +133,16 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a symmetric decryptor object with the current <see cref="P:System.Security.Cryptography.SymmetricAlgorithm.Key" /> property and initialization vector (<see cref="P:System.Security.Cryptography.SymmetricAlgorithm.IV" />).</summary>
+        <returns>A symmetric decryptor object.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ This method decrypts an encrypted message created using the <xref:System.Security.Cryptography.SymmetricAlgorithm.CreateEncryptor%2A> overload with the same signature.  
```

Comment:
Should this use DESCryptoServiceProvider instead? Also linking to the same overload.

```suggestion
 This method decrypts an encrypted message created using the <xref:System.Security.Cryptography.DESCryptoServiceProvider.CreateEncryptor> overload with the same signature.  
```